tla-pp/src/main/scala/at/forsyte/apalache/tla/pp/passes/ConfigurationPassImpl.scala

```diff
@@ -87,13 +85,12 @@ class ConfigurationPassImpl @Inject()(val options: WriteablePassOptions,
   }
 
   // copy the relevant options
-  private def copyRelevantOptions(): WriteablePassOptions = {
+  private def copyRelevantOptions(from: PassOptions, to: WriteablePassOptions): Unit = {
```

Comment:
Nice, yeah this definitely helps make the flow of information a bit clearer. Thanks :)